xbmc/addons/kodi-addon-dev-kit/include/kodi/gui/gl/GLonDX.h

```diff
@@ -0,0 +1,369 @@
+/*

```

Comment:
replace the windows line ending with an unix line ending in this line

Reply:
Such a crap, still one there :smile:

Reply:
A question to prevent this CRLF problem, makes it sense to create a request where a `.gitattributes` with `text eol=lf` becomes added to Kodi?

https://help.github.com/en/articles/dealing-with-line-endings